client/shaders/second_stage/opengl_fragment.glsl

```diff
@@ -86,6 +95,7 @@ void main(void)
 	{
 #if ENABLE_TONE_MAPPING
 		color = applyToneMapping(color);
+		color.rgb = applySaturation(color.rgb, saturation);
```

Comment:
As discussed let's move this out of ENABLE_TONE_MAPPING. I'll prepare a pull request to add Reinhard TMO, so we don't have the weird boolean flag.

Reply:
Makes sense.
Would we need a separate switch '#if SATURATION` then?

Reply:
What would be the rationale? Do you think optimization?

Reply:
We have switches for everything else. :)  Yeah, for performance.
Not that I like the switches.

Maybe we should keep ENABLE_TONE_MAPPING as general switch, and do all specific tone mapping guarded by it.
Or we get rid of those switches altogether and always have these enabled... I think that's the route you favor.